core/src/popover/popover.tsx

```diff
@@ -12,9 +12,21 @@ interface ContentProps {
 }
 
 export interface PopoverProps {
+	/**
+	 * The component that will be render Popover when action on it
+	 */
 	target: (props: TargetProps) => React.ReactNode;
+	/**
+	 * The elements to be displayed within the Popover
+	 */
 	content: (props: ContentProps) => React.ReactNode;
+	/**
+	 * `target`'s wrapper
+	 */
 	TargetWrapper?: () => JSX.Element;
+	/**
+	 * Describes the preferred placement of the Popover relative to the `target`
+	 */
```

Comment:
Should mention and have a link to Popper's placement, so our users know what is the expected strings https://popper.js.org/docs/v2/constructors//#options